{kokoro} Website generation is broken

```
In file included from ../src/binding.cpp:1:
In file included from ../../nan/nan.h:191:
../../nan/nan_maybe_43_inl.h:112:15: error: no member named 'ForceSet' in 'v8::Object'
  return obj->ForceSet(isolate->GetCurrentContext(), key, value, attribs);
         ~~~  ^
In file included from ../src/binding.cpp:1:
In file included from ../../nan/nan.h:197:
In file included from ../../nan/nan_new.h:189:
../../nan/nan_implementation_12_inl.h:337:28: warning: 'New' is deprecated: Use Isolate* version [-Wdeprecated-declarations]
  return v8::StringObject::New(value).As<v8::StringObject>();
                           ^
/Users/kbuilder/.node-gyp/11.7.0/include/node/v8.h:5242:37: note: 'New' has been explicitly marked deprecated here
                static Local<Value> New(Local<String> value));
```

<!-- Auto-generated content below, do not modify -->

---
#### Internal data
- Associated internal bug: [b/123242943](http://b/123242943)

--- a/.kokoro
+++ b/.kokoro
@@ -426,6 +426,13 @@ generate_website() {
 
     gem_install bundler
     brew_install yarn --without-node
+    
+    # The above installations update node to v10, but the docsite generator
+    # relies on v8.
+    brew unlink node
+    brew install node@8
+    # Required because node@8 is keg-only, meaning it isn't linked automatically.
+    brew link --force --overwrite node@8
   fi
 
   cd docsite-generator

